### PR TITLE
test: upgrade jest

### PR DESCRIPTION
**Pull request by andersfylling**
upgrading jest allows us to setup retries within a specific file. This should solve the issue with flaky integration tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,13 +59,13 @@
     "eslint-plugin-lodash": "^7.1.0",
     "eslint-plugin-prettier": "^3.1.4",
     "husky": "^3.0.2",
-    "jest": "^26.1",
+    "jest": "^27.0.0",
```
Comment (ebrattli) [inline package.json]: the latest version is `27.0.6`

Comment (ebrattli) [reply package.json]:
```suggestion
    "jest": "^27.0.6",
```

File: package.json [hunk 2]
```diff
@@ -59,13 +59,13 @@
...
     "lerna": "^3.22.1",
     "markdown-link-check": "^3.8.1",
     "nock": "^10.0.6",
     "prettier": "^1.10.2",
     "rollup": "^2.22.2",
-    "ts-jest": "^26.1",
+    "ts-jest": "^27.0.0",
```
Comment (ebrattli) [inline package.json]:
```suggestion
    "ts-jest": "^27.0.5",
```

**Comment by andersfylling** [comment]
looks like upgrading causes some issues we need to patch up
image

**Comment by andersfylling** [comment]
closing this as I've gotten too busy.

**Pull request closed without merging**